src/utils/hasConfigurableGlobal.test.ts

```diff
@@ -26,6 +26,15 @@ it('returns false if the global property does not exist', () => {
   expect(hasConfigurableGlobal('_non-existing')).toBe(false)
 })
 
+it('returns false if the global property exists and it\'s value is undefined', () => {
+  Object.defineProperty(global, '_existsAndUndefined', {
+    value: undefined,
+    configurable: true,
+  })
+
+  expect(hasConfigurableGlobal('_existsAndUndefined')).toBe(true)
```

Comment:
This has to be `false` though. Proxying `undefined` will throw an error as it's not an object. 

You are correctly returning `false` in the implementation, but the test still expects `true`. Can you adjust the assertion please?

Reply:
You are right!